src/client/package.json

```diff
@@ -57,12 +57,17 @@
   "devDependencies": {
     "autoprefixer": "6.3.1",
     "autoprefixer-loader": "3.1.0",
-    "babel": "5.8.23",
-    "babel-core": "5.8.33",
+    "babel": "6.5.2",
```

Comment:
yeah, I know - just asking for re-generation of the shrinkwrap - by default `npm shrinkwrap` is meant for prod so it skips `devDependencies` unless you pass on the --dev flag. I did pass it before committing as the goal is to be able to produce a reliable build but checking if this is how you had it before.